lib/mix_unused/analyzers/private.ex

```diff
@@ -7,7 +7,7 @@ defmodule MixUnused.Analyzers.Private do
   def message, do: "should be private (is not used outside defining module)"
 
   @impl true
-  def analyze(data, all_functions) do
+  def analyze(data, all_functions, _config \\ nil) do
```

Comment:
```suggestion
  def analyze(data, all_functions, _config) do
```

Reply:
This was just to leave all the tests unchanged, but ok I can remove the default value.